root/statistics/LanguagesScripts.js

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2018 Shamroy Pellew
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+const manifest = require('../static/manifest');
+const {l} = require('../static/scripts/common/i18n');
+const Layout = require('./Layout');
+
+const formatCount = (n) => {
+  return n.toLocaleString($c.stash.current_language);
+};
+
+const LinkSearchableProperty = ({
+  searchField,
+  searchValue,
+  entityType,
+  text,
+}) => {
+  const params = {limit: 25, method: 'advanced', query: searchField + ':\"' + searchValue + '\"', type: entityType};
+  return (
+    <a href={'/search?' + Object.keys(params).map(key => key + '=' + params[key]).join('&')}>
+      {text ? text : searchValue}
+    </a>
+  );
+};
+
+const LanguagesScripts = () => (
+  <Layout fullWidth page="languages-scripts" title={l('Languages and Scripts')}>
+    {/* [%- PROCESS "statistics/macros-header.tt" -%] */}
```

Comment:
Replace all of these (the macros-header ones, not macros-footer) with just `{manifest.css('statistics')}`.

The only other thing macros-header.tt did was switch gettext domains, and that's related to my comment at https://github.com/metabrainz/musicbrainz-server/pull/609/files#r171417713 which I said I could look at later.